Gradle.md

```diff
@@ -8,6 +8,8 @@ There are currently five experiments for Gradle. You could also perform these ex
 
 ## Requirements
 
+You must have [Develocity](https://gradle.com/develocity/) in order to use the Develocity Build Validation Scripts. At a minimum, your Develocity user must have the permissions `Publish build scans` and `View build scans and build data` granted explicitly or anonymously, as analyzing the results relies on the use of build scans.
```

Comment:
This is correct, although I wonder if we should mention the benefit of having API here as well

Reply:
I thought about that, but ultimately decided not to since it isn't actually a "requirement". We also mention the API [here](https://github.com/gradle/develocity-build-validation-scripts/blob/main/Gradle.md#authenticating-with-develocity).

Reply:
```suggestion
You must have [Develocity](https://gradle.com/develocity/) in order to use the Develocity Build Validation Scripts. At a minimum, your Develocity user must have the permissions `Publish build scans` and `View build scans and build data` granted explicitly or anonymously, as analyzing the results relies on the use of build scans. To get the full benefit of the Develocity Build Validation Scripts, your Develocity user should also be granted the `Access build data via the API` permission.
```

I think it's okay to mention the API permission here as well. It increases the probability that the reader will actually see it.

Reply:
If we call out API as a requirement then I feel we should do the same for CCUD. Both are optional. Do you think it should also be included here under "requirements"? It also has its own section already later down in the README.

Reply:
> If we call out API as a requirement then I feel we should do the same for CCUD.

Good call. I think that's a good idea as well. Then there is just one place where the user can a list of everything they may need.

The sections further down go into more detail, so maybe we can briefly mention the API permission and CCUD here, and link to the later sections.